### ISSUE TITLE
Gradient explosion

### ISSUE THREAD

**Issue by tiendat104** [DevinKreuzer/SAN#4]
Hello, I wonder whether there exists any training setting of your model that experience gradient explosion at the early stage of training. 
I have included LPE embedding to my code several days ago and train it for the dataset pcqm4m. Unfortunately, the gradient explodes dramatically during the early training. If I turn off LPE embedding, then gradient explosion will not happen. I wonder whether LPE is prone to gradient explosion or not and if so, is there anyway to solve to issue?
Thank you.

**Comment by DevinKreuzer** [comment]
Hi,

Yes, I noticed this as well on a few instances. I never managed to isolate the issue, though. Have you tried smaller learning rates?

Devin
